### ISSUE TITLE
when a quest starts, a `system` message should be written to party chat

### ISSUE THREAD

**Issue by Alys** [HabitRPG/habitica#8072]
When a quest starts (i.e., after the invitation phase is finished) a message should be posted to party chat saying something like:
`Your quest, The Call of Octothulu, has started.`

It should  be posted by the system user, as for other automated quest messages.

This would help players be aware of when they need to watch their Dailies carefully, and would help admins with troubleshooting quest bugs.

In addition, I'd like a new `info` field to be added to the system message, listing the UIDs of the participants (see below for an example). This is solely to help with troubleshooting for completed quests so it doesn't need to be visible to the players or accessible through the API (unless any other admins disagree). I've chosen `info` as the field name so that in future it can be used for other purposes too.

```
        {
            "id": "2498af5a-cf78-4e22-93b4-3f41d79f517c",
            "text": "`Your quest, The Call of Octothulu, has started.`",
            "info": "participants: 9264741a-e7df-4202-8eaf-5e32bba01f1c, f58f8ff8-8a7f-4a49-8cda-789354b5be92, 7b55f43a-49b5-4f05-bff1-fb3994311dc0",
            "timestamp": 1474102683475,
            "likes": {},
            "flags": {},
            "flagCount": 0,
            "uuid": "system"
        },
```

**Comment by lemoness** [comment]
This looks good to me! Go ahead and mark it "help welcome."

**Comment by dumindux** [comment]
Hi, I would like to work on this issue. Its great if you can point me where to get started. Thanks.

**Comment by NicoSiebler** [comment]
@dumindux I already tried to solve this for the first part of the issue in a short time without luck yet and I didn't mention this as this was my first task try. So I hope this could help:
My guess was to adjust the `startQuest()` method in **server/models/group.js** directly above the `sendTxnEmail()` with a `this.sendChat()` but the object was missing a usable chat to send the message and I didn't look deep enough what I did wrong or what was missing. Maybe it's simply the wrong spot or approach. I would be interested in the proper solution.
